static/skywire-manager-src/src/app/components/vpn/pages/vpn-settings/vpn-settings.component.ts

```diff
@@ -0,0 +1,233 @@
+import { Component, OnDestroy, ViewChild } from '@angular/core';
+import { Subscription } from 'rxjs';
+import { ActivatedRoute } from '@angular/router';
+import { MatDialog } from '@angular/material/dialog';
+
+import { VpnHelpers } from '../../vpn-helpers';
+import { BackendState, VpnClientService, VpnServiceStates } from 'src/app/services/vpn-client.service';
+import { SnackbarService } from 'src/app/services/snackbar.service';
+import { AppsService } from 'src/app/services/apps.service';
+import { processServiceError } from 'src/app/utils/errors';
+import { DataUnits, VpnSavedDataService } from 'src/app/services/vpn-saved-data.service';
+import GeneralUtils from 'src/app/utils/generalUtils';
+import { SelectableOption, SelectOptionComponent } from 'src/app/components/layout/select-option/select-option.component';
+import { TopBarComponent } from 'src/app/components/layout/top-bar/top-bar.component';
+
+/**
+ * Options that VpnSettingsComponent might be changing asynchronously.
+ */
+enum WorkingOptions {
+  None = 0,
+  Killswitch = 1,
+}
+
+/**
+ * Page for changing the configuration of the VPN client.
+ */
+@Component({
+  selector: 'app-vpn-settings-list',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/skycoin/skywire/pullRequest?prid=7163595)
```suggestion
  selector: "app-vpn-settings-list",
```